config/Config-kernel.in

```diff
@@ -1168,6 +1174,13 @@ if KERNEL_IPV6
 
 endif
 
+	config KERNEL_MPTCP_IPV6
+		bool "Enable IPv6 MultiPath TCP support"
+		default n
+		depends on KERNEL_MPTCP
+		help
+		  IPv6 MultiPath TCP to kernel support.
```

Comment:
Hi!
This should be above the endif for "if KERNEL_IPV6", right?.

Reply:
Sorry, my mistake. Fixed.